__tests__/integration.test.js

```diff
@@ -132,20 +132,20 @@ it('should correctly return array of objects with a status 200 code', () => {
     });
   })
 });
-it('should return 404 when passed an article id that does not exist', () => {
```

Comment:
this test was correct as it was before - Ali meant if the article id exists but it doesnt actually have any comments - we want a new test that tests this scenario (we dont want to give an error if the article doesnt have any comments, we should just return an empty array